mappings/net/minecraft/world/gen/chunk/NoiseChunkGenerator.mapping

```diff
@@ -31,6 +35,16 @@ CLASS net/minecraft/unmapped/C_yjuvgdyy net/minecraft/world/gen/chunk/NoiseChunk
 		ARG 4 chunk
 	METHOD m_gmkezvqm (Lnet/minecraft/unmapped/C_yjuvgdyy;)Lnet/minecraft/unmapped/C_ajguowya;
 		ARG 0 generator
+	METHOD m_hfhrpedc getDebugPreliminarySurfaceLevelBlockState (Lnet/minecraft/unmapped/C_rvjywqnu;IIILnet/minecraft/unmapped/C_txtbiemp;)Lnet/minecraft/unmapped/C_txtbiemp;
```

Comment:
Maybe just put getPreliminarySurfaceLevelBlockState instead? I don't see a need for the Debug.

Reply:
so I presume it would just be better to change the name then? there isn't any other method that gets the blockstate from the preliminary surface from what im seeing